docs/reference/content/builders/aggregation.md

```diff
@@ -10,8 +10,9 @@ title = "Aggregates"
 
 ## Aggregates
 
-The [`Aggregates`]({{< apiref "mongodb-driver-core" "com/mongodb/client/model/Aggregates" >}}) class provides static factory methods that build [aggregation
-pipeline operators]({{< docsref "reference/operator/aggregation/" >}}).  Each method returns an instance of the
+The [`Aggregates`]({{< apiref "mongodb-driver-core" "com/mongodb/client/model/Aggregates" >}}) class provides static factory methods
+that build [aggregation pipeline stages]({{< docsref "meta/aggregation-quick-reference/#stages" >}}).
```

Comment:
The link to aggregation pipeline stages was incorrect. Also filed https://jira.mongodb.org/browse/DOCSP-17428 to fix the link in the new Java driver docs.